dissect/target/plugins/filesystem/ntfs/mft.py

```diff
@@ -151,7 +152,7 @@ def check_compatible(self) -> None:
         "--macb",
         group="fmt",
         action="store_true",
-        help="compacts the MFT entry timestamps into aggregated records with MACB bitfield",
+        help="compacts MFT timestamps into MACB bitfield (format: standard|ads / filename)",
```

Comment:
Do you think this maybe makes the format more explanatory?
```suggestion
        help="compacts MFT timestamps into MACB bitfield (format: MACB[standard|ads]/MACB[filename])",
```